tests/common/compact_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.etcd.io/etcd/tests/v3/framework/config"
+	"go.etcd.io/etcd/tests/v3/framework/testutils"
+)
+
+func TestCompact(t *testing.T) {
+
+	testRunner.BeforeTest(t)
+	tcs := []struct {
+		name    string
+		options config.CompactOption
+	}{
+		{
+			name:    "CompactNoPhysical",
```

Comment:
```suggestion
			name:    "NoPhysical",
```

No need to repeat "Compact" as name of the test function `TestCompact` will be used as a prefix. So it will be `TestCompact/CompactNoPhysical`.